Repository/RevisionRepository.php

```diff
@@ -628,4 +661,27 @@ public function findAllLockedRevisions(ContentType $contentType, string $lockBy,
 
         return new Paginator($qb->getQuery());
     }
+
+    public function findDraftsByContentType(ContentType $contentType): array
+    {
+        return $this->findBy([
```

Comment:
I don't like the magic doctrine findBy function, it creates a depend that we don't want.
Can you write the query using a queryBuilder